docker/lib/dependabot/docker/tag.rb

```diff
@@ -89,7 +89,7 @@ def version
 
       def format
         return :year_month if version.match?(/^[12]\d{3}(?:[.\-]|$)/)
-        return :year_month_day if version.match?(/^[12]\d{5}(?:[.\-]|$)/)
+        return :year_month_day if version.match?(/^[12]\d{5,7}(?:[.\-]|$)/)
```

Comment:
This will match 5, 6, or 7 digits. Should we explicitly match 5 or 7 instead, since 6 digits (plus a 1 or 2) would be an invalid date. Something like this:

```suggestion
        return :year_month_day if version.match?(/^[12](?:\d{5}|\d{7})(?:[.\-]|$)/)
```